src/Features/Core/Portable/AddImport/References/PackageReference.cs

```diff
@@ -28,10 +29,10 @@ private partial class PackageReference : Reference
             }
 
             public override async Task<AddImportFixData> TryGetFixDataAsync(
-                Document document, SyntaxNode node, bool placeSystemNamespaceFirst, CancellationToken cancellationToken)
+                Document document, SyntaxNode node, bool placeSystemNamespaceFirst, AddImportPlacement placement, CancellationToken cancellationToken)
```

Comment:
consider an AddImportOptions struct that can pass along the placement and the PlaceSystemNamespaceFirst value.